src/test/java/codeu/controller/ActivityFeedServletTest.java

```diff
@@ -21,17 +21,17 @@
 
   @Before
   public void setup() throws IOException {
-    activityfeedServlet = new ActivityFeedServlet(); 
-    mockRequest = Mockito.mock(HttpServletRequest.class);
-    mockResponse = Mockito.mock(HttpServletResponse.class);
-    mockOutputStream = Mockito.mock(ServletOutputStream.class);
-    Mockito.when(mockResponse.getOutputStream())
-    .thenReturn(mockOutputStream);
+  activityfeedServlet = new ActivityFeedServlet(); 
```

Comment:
Revert all these changes? I think the indents were right before

Reply:
How do I revert a change